### PR TITLE
Added Hive SLR2 and SLT3 to devices.js

### PR DESCRIPTION
**Pull request by aliijm**
Added Hive SLR2 and SLT3 to devices.js (see [zigbee2mqtt#4540](https://github.com/Koenkk/zigbee2mqtt/issues/4540)).

The converters for the thermostat clusters don't seem to currently support multiple endpoints. This support would need to be added to allow the the SLR2 to function correctly, but I don't know how to do this.

### CODE REVIEW

**Comment by Koenkk** [comment]
I've made some updates to fromZigbee.js to support multiEndpoint, can you check if it works OK now?

File: converters/toZigbee.js [hunk 1]
```diff
@@ -968,11 +968,17 @@ const converters = {
     thermostat_occupied_heating_setpoint: {
         key: ['occupied_heating_setpoint'],
         convertSet: async (entity, key, value, meta) => {
-            const occupiedHeatingSetpoint = (Math.round((value * 2).toFixed(1)) / 2).toFixed(1) * 100;
```
Comment (Koenkk) [inline converters/toZigbee.js:971]: Changes to toZigbee shouldn't be needed, the correct endpoint (entity) will be passed when using `zigbee2mqtt/[THERMOSTAT]/heat/set`/`zigbee2mqtt/[THERMOSTAT]/cool/set` as topic

Comment (aliijm) [reply converters/toZigbee.js:971]: The reports from the thermostat are working, thanks. When I try to publish a command using `zigbee2mqtt/[THERMOSTAT]/heat/set` I get `Entity '[THERMOSTAT]/heat' is unknown`

Comment (Koenkk) [reply converters/toZigbee.js:971]: Ah sorry, the endpoint names should be one of: https://github.com/Koenkk/zigbee2mqtt/blob/master/lib/util/utils.js#L31

Comment (aliijm) [reply converters/toZigbee.js:971]: When I add the endpoint names to that file it all works perfectly (without my changes to toZigbee). Should I submit a pr to zigbee2mqtt to add the heat and water endpoints? Thanks so much for your help!

Comment (Koenkk) [reply converters/toZigbee.js]: No, I will add it.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**